packages/fsweb/proxy/proxy.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const proxy = require('http-proxy-middleware');
+const url = require('url');
+const { removeDWCookies, mutateDWCookies } = require('./lib');
```

Comment:
We should make the Demandware-specific code an option that you can turn on, so the proxy can support other apis.

Reply:
@Cauldrath done